cdap-app-templates/cdap-etl/cdap-etl-proto/src/main/java/io/cdap/cdap/etl/proto/v2/ETLBatchConfig.java

```diff
@@ -181,9 +204,12 @@ public static ETLBatchConfig forSystemService() {
     private Engine engine;
     private List<ETLStage> endingActions;
     private Integer maxConcurrentRuns;
+    // Only used for upgrade purpose.
+    private List<String> comments;
```

Comment:
don't need this, can just always pass Collections.emptyList() in the build() method